tests/fsharp/core/versionAttributes/out.stderr.bsl

```diff
@@ -0,0 +1,30 @@
+Test 1=================================================
+
+NoWarn2003.fs(1,1): warning FS0988: Main module of program is empty: nothing will happen when it is run
+Test 2=================================================
+
+NoWarn2003_2.fs(8,6): warning FS0988: Main module of program is empty: nothing will happen when it is run
+Test 3=================================================
+
+Warn2003_1.fs(5,42): warning FS2003: An System.Reflection.AssemblyInformationalVersionAttribute specified version '5.0.0-beta024', but this value is invalid and has been ignored
+
+Warn2003_1.fs(7,6): warning FS0988: Main module of program is empty: nothing will happen when it is run
+Test 4=================================================
+
+Warn2003_2.fs(4,42): warning FS2003: An System.Reflection.AssemblyInformationalVersionAttribute specified version '5.0.0-beta024', but this value is invalid and has been ignored
+
+Warn2003_2.fs(6,6): warning FS0988: Main module of program is empty: nothing will happen when it is run
+Test 5=================================================
+
+Warn2003_3.fs(4,38): warning FS2003: An System.Reflection.AssemblyVersionAttribute specified version '5.0.0-beta024', but this value is invalid and has been ignored
```

Comment:
The use of `An` in the error message looks wrong.  Perhaps change to `The attribute ....`

Reply:
Did I change this or is it something that makes sense changing (just curious)

Reply:
@dsyme Just to clarify: I change all messages to
`arning FS2003: The attribute System.Reflection.AssemblyInformationalVersionAttribute specified version '5.0.0-beta024', but this value is invalid and has been ignored`, correct?

Reply:
great, thanks

Reply:
ok done, lets see if it is still green...